ARrehab/ARrehab/PassThroughView.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  File.swift
+//  ARrehab
+//
+//  Created by Eric Wang on 5/2/20.
+//  Copyright © 2020 Eric Wang. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+/**
+ Clear UIView that allows touch input to be passed through it to background views.
+ */
+class PassThroughView: UIView {
```

Comment:
Could you explain the functionality of this? Like why the user would have to interact with a UIView that is one or more layers beneath the top one?

Reply:
this is currently used for development purposes where my minigame switch is in the main ViewController (soon to be replaced with in game logic triggering and deactiviating the minigame). However whatever persistent UI between the gameboard and the Minigame most likely will reside within the Main ViewController.

Not really sure how to implement the loading of the MinigameViewController yet but one thought was just to overlay it across the whole screen (hence the need to interact with the persistent UI beneath).

Another option is to load it into a subset of the screen (I believe the current implementation is the subset of the screen that is shifted 500 px down) however this decreases screen real estate...